sparse/src/KokkosSparse_spmv_handle.hpp

```diff
@@ -351,6 +356,7 @@ struct SPMVHandle
     } else {
       switch (get_algorithm()) {
         case SPMV_MERGE_PATH:
```

Comment:
Actually is there no TPL that implements a merge path like algorithm for Bsr?

Reply:
Nope, rocsparse_xbsrmv, cusparseXbsrmv/cusparseXbsrmm, and mkl_sparse_x_mm/mkl_sparse_x_mm all do not have an algorithm parameter at all. So unless one of them changes their interface (or we add BSR merge path) there is no special path for imbalanced matrices in native or any TPL.

Reply:
Thanks for checking!